generators/client/templates/angular/src/main/webapp/app/core/auth/principal.service.ts.ejs

```diff
@@ -90,6 +98,14 @@ export class Principal {
                 <%_ if (websocket === 'spring-websocket') { _%>
                 this.trackerService.connect();
                 <%_ } _%>
+
+                <%_ if (enableTranslation && authenticationType !== 'oauth2') { _%>
+                // After retrieve the account info, the language will be changed to
+                // the user's preferred language configured in the account setting
+                const langKey = this.sessionStorage.retrieve('local') || this.userIdentity.langKey;
```

Comment:
the key name is very misleading, I think we should change it to something like `language` or `locale`

Reply:
yeah, spell error